Fix typos; make minor edits (#1)
salesforce/oss-template: CODE_OF_CONDUCT.md

community fosters innovation and creativity, and are committed to building a
culture where everyone feels included.

> Salesforce open-source projects are committed to providing a friendly, safe, and

Reviewer: ```suggestion
Salesforce open source projects are committed to providing a friendly, safe, and
```

It also establishes a mechanism for reporting issues and resolving conflicts.

All questions and reports of abusive, harassing, or otherwise unacceptable behavior

> in a Salesforce open-source project may be reported by contacting the Salesforce

Reviewer: ```suggestion
in a Salesforce open source project may be reported by contacting the Salesforce
```

* Being respectful of differing viewpoints and experiences
* Gracefully accepting constructive criticism
* Focusing on what is best for the community
* Showing empathy toward other community members

Examples of unacceptable behavior by participants include:

* The use of sexualized language or imagery and unwelcome sexual attention or
advances
* Personal attacks, insulting/derogatory comments, or trolling
* Public or private harassment

> * Publishing, or threatening to publish, others’ private information—such as

Reviewer: ```suggestion
* Publishing, or threatening to publish, others' private information—such as
```

Project maintainers who do not follow or enforce the Code of Conduct in good
faith may face temporary or permanent repercussions as determined by other

> members of the project’s leadership and the Salesforce Open Source Conduct 

Reviewer: ```suggestion
members of the project's leadership and the Salesforce Open Source Conduct 
```
